netbox/dcim/forms/model_forms.py

```diff
@@ -1396,6 +1396,7 @@ class InterfaceForm(InterfaceCommonForm, ModularDeviceComponentForm):
             'rf_role', 'rf_channel', 'rf_channel_frequency', 'rf_channel_width', 'wireless_lan_group', 'wireless_lans',
             name=_('Wireless')
         ),
+        FieldSet('vlan_translation_policy', name=_('VLAN Translation'))
```

Comment:
This needs to be added to VMInterface form as well, and `vlan_translation_policy` needs to be defined as a DynamicModelChoiceField.